### PR TITLE
Documents for Docker-compose

### LINKED ISSUE #989: [TRACKING ISSUE] Docs 1.0
We sorely need better HospitalRun docs. Currently, we don't have any end user docs, and our developer documentation is limited.

This issue is for high level discussion and planning for our overall docs content strategy. All PR's and other issues that relate to Docs should reference this issue, and be referenced in this issue for tracking purposes.

## Scope and stages

To properly scope work on our documentation, here's a first pass at a phasing outline:

### 1. Make developer docs more comprehensive

Currently we have some basic developer documentation in the README, Contributing guide, and Wiki. But we need a more comprehensive set of developer docs. Here's a basic outline to get us started:

- [ ] Getting Started
  - [ ] Setting up your local environment
  - [ ] Ember basics for HospitalRun
- [ ] App architecture
  - [ ] Routes
  - [ ] Templates
  - [ ] Components
  - [ ] Controllers
  - [ ] Models
- [ ] Frontend
- [ ] Server
- [ ] Styleguide
  - [ ] Ember/JavaScript
  - [ ] SCSS
  - [ ] Pattern library of UI elements
- [ ] Tests

(@HospitalRun/core-maintainers feel free to edit this outline)

### 2. Publish developer docs

We need to get the developer docs into a centralized, easily accessible location. Right now they are spread apart across the [README](https://github.com/HospitalRun/hospitalrun-frontend/blob/master/README.md), [CONTRIBUTING](https://github.com/HospitalRun/hospitalrun-frontend/blob/master/.github/CONTRIBUTING.md) doc, and [wiki](https://github.com/HospitalRun/hospitalrun-frontend/wiki).

Let's get them centralized onto the Docs site. It would be great if we had a way to include them both in the frontend repo, and on the docs site (hospitalrun.io). Thinking we might be able to accomplish that with a separate repo called `/docs`, and just include that as a git submodule in both the app, and the marketing site. Thoughts on that approach?

### 3. Write end user help docs

Finally, we need a good story for end user help documentation. Ultimately I think that should be accessible directly from the application, but might also be helpful to have that available at hospitalrun.io as well. If the submodule approach works well for developer docs, perhaps we could use that approach for end user help docs too?

## Where help is needed:

We're going to need a bunch of help with this effort. Here's the shortlist that I can identify (again, @HospitalRun/core-maintainers please contribute to this list where I've left anything out, or where it could be clearer):

- [ ] we need a finalized content outline for developer docs
- [ ] we need a rough content outline for end user help docs _(perhaps @mnorbeck might be interested to take a first pass at this one?)_
- [ ] we need people to start picking sections from the above outlined docs and spike them into full articles
- [ ] we need someone to investigate and possibly setup a `/docs` and `/user-help` repo as git submodules
- [ ] we'll need design for how the docs will appear on the hospitalrun.io site
- [ ] we'll need design for how the docs will appear in the app
- [ ] we should start breaking this tracking issue out into smaller scoped issues that we can reference from here

---

cc @HospitalRun/core-maintainers

### PR DESCRIPTION
**Pull request by Anisha1234**
Fixes #989 
**Changes proposed in this pull request:**
- resolves the conflicts between README.md and contribution guide about the process of installation
- adds necessary setup instructions regarding docker-compose up. 

cc @jkleinsc @jglovier @tangollama

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,10 +36,11 @@ To install the frontend please do the following:
 3. Install [ember-cli latest](https://www.npmjs.org/package/ember-cli): `npm install -g ember-cli@latest`.
    Depending on your [npm permissions](https://docs.npmjs.com/getting-started/fixing-npm-permissions) you might need root access to install ember-cli.
 4. Install [bower](https://www.npmjs.org/package/bower): `npm install -g bower`
-5. Clone this repo with `git clone https://github.com/HospitalRun/hospitalrun-frontend`, go to the cloned folder and run `script/bootstrap`.
+5. Fork the master repository from `https://github.com/HospitalRun/hospitalrun-frontend` and clone the forked repository.
```
Comment (jkleinsc) [inline README.md]: I'm not sure why the change here.  For some folks cloning is all they will need to do.

File: README.md [hunk 2]
```diff
@@ -75,13 +76,16 @@ To run HospitalRun with Docker please do the following:
 - Build the HospitalRun image with `docker build -t hospitalrun-frontend .`
 - Execute `docker run -it --name couchdb -d couchdb` to create the couchdb container.
 - Execute `docker run -it --name hospitalrun-frontend -p 4200:4200 --link couchdb:couchdb -d hospitalrun-frontend` to create the HospitalRun container.
+or instead,
```
Comment (jkleinsc) [inline README.md]: The way this is formatted it isn't immediately clear that it is a different way to accomplish Running with Docker.  I suggest breaking the Running With Docker into two sub sections, eg:
##Running With Docker
###Running With Docker Engine
Existing information here
###Running with Docker Compose
Your new info here

**Comment by Anisha1234** [comment]
@jkleinsc I have changed the doc according  to your  suggestion.Kindly have a look at it
